lib/resty/t1k/request.lua

```diff
@@ -310,6 +314,7 @@ function _M.do_request(opts)
         ngx.header.content_type = consts.BLOCK_CONTENT_TYPE
         local response = fmt(consts.BLOCK_CONTENT_FORMAT, rsp_code, event_id)
 
+        -- no need to set ngx.status, ngx.say or ngx.exit, let the caller handle it
```

Comment:
is it okay to have a separate `handler` module called by the `t1k.do_access()` function? 

see https://github.com/chaitin/lua-resty-t1k/pull/4/files#diff-f2f73c714951bb76ea9093b420cf5e423d764914564cd573323d7d6f64c031c7R79